com.spotify.Client.json

```diff
@@ -150,6 +150,88 @@
                 }
             ]
         },
+        /* THESE CANNOT BE UPDATED UNTIL SPOTIFY SUPPORTS ffmpeg 5 ------------------------------------------------------- */
```

Comment:
Does f-e-d-c preserve comments like this?

Reply:
No idea what f-e-d-c is could you explain?

Reply:
Spotify uses [this](https://github.com/flathub/flatpak-external-data-checker) project to keep dependencies updated. It rewrites manifest in order to do that and I'm not sure if it preserves comments in json format while doing so.

Reply:
oh external data checker. Yeah i believe it does

Reply:
But the ffmpeg and sdl2 dont have external data checker and we cant add them as newer versions create issues

Reply:
I know but f-e-d-c lints whole manifest

Reply:
only way to be sure is to wait for a f-e-d-e-c to run. It wont commit automatically (as far as i am aware).
If it does we will just have to add them back.
i dont think it does though as i have seen other manifests with the comments

Reply:
You can check locally: https://github.com/flathub/flatpak-external-data-checker#installation

Reply:
Doing that now

Reply:
Yeah it is fine with the comment and has left it in place

Reply:
Ok, thx for testing

Reply:
Just for future reference:
```flatpak run --filesystem=~ org.flathub.flatpak-external-data-checker /home/a/Documents/com.spotify.Client/com.spotify.Client.json
INFO    src.manifest: Checking 10 external data items
INFO    src.manifest: Started check [1/10] archive libnotify/libnotify-0.8.1.tar.xz (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [2/10] archive xprop/xprop-1.2.5.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [3/10] archive python-setuptools_scm/setuptools_scm-6.0.1.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [4/10] archive python-xlib/python-xlib-0.31.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Started check [5/10] git spotifywm/spotifywm.git (from com.spotify.Client.json)
INFO    src.checkers.gitchecker: Skipping source spotifywm.git, commit is specified, but neither tag nor branch
INFO    src.manifest: Finished check [4/10] git spotifywm/spotifywm.git (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [6/10] archive lzo/lzo-2.10.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Started check [7/10] git squashfs-tools/squashfs-tools.git (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [8/10] archive sdl2/SDL2-2.0.22.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Skipped check [9/10] archive ffmpeg-4.4/ffmpeg-4.4.2.tar.gz (from com.spotify.Client.json)
INFO    src.manifest: Started check [10/10] extra-data spotify/spotify.snap (from com.spotify.Client.json)
INFO    src.manifest: Finished check [8/10] git squashfs-tools/squashfs-tools.git (from com.spotify.Client.json)
INFO    src.manifest: Finished check [9/10] archive libnotify/libnotify-0.8.1.tar.xz (from com.spotify.Client.json)
INFO    src.manifest: Finished check [10/10] extra-data spotify/spotify.snap (from com.spotify.Client.json)
INFO    src.main: Check finished with 0 error(s)```